cmd/transfer-pvc/transfer-pvc.go

```diff
@@ -502,6 +598,18 @@ func createAndWaitForRoute(pvc *corev1.PersistentVolumeClaim, destClient client.
 	return e
 }
 
+// getDestinationPVC given a source PVC, returns a PVC to be created in the destination cluster
+func (t *TransferPVCOptions) getDestinationPVC(sourcePVC *corev1.PersistentVolumeClaim) *corev1.PersistentVolumeClaim {
+	pvc := sourcePVC.DeepCopy()
+	clearDestPVC(pvc)
```

Comment:
looks like this is only used here in `getDestinationPVC`, can we merge what's in that function with what you have below?